scss/styles.scss

```diff
@@ -1,22 +1,22 @@
-// Libraries 
-@import "lib/bourbon/_bourbon.scss";
-@import "lib/neat/_neat.scss";
-@import "fonts";
+// Libraries
+@import 'lib/bourbon/_bourbon';
+@import 'lib/neat/_neat';
+@import 'fonts';
 
 // Variables and mixins
-@import "variables";
-@import "grid";
-@import "mixins/_utilities.scss";
-@import "mixins/_type-mixins.scss";
+@import 'variables';
+@import 'grid';
+@import 'mixins/_utilities';
+@import 'mixins/_type-mixins';
```

Comment:
Imported partial `mixins/_type-mixins` should be written as `mixins/type-mixins`